apps/web/src/app/[locale]/(dashboard)/[wsId]/inventory/products/[productId]/form.tsx

```
@@ -341,121 +316,213 @@
-                      />
+                                  >
+                                    <SelectValue
+                                      placeholder={t(
+                                        'ws-inventory-units.placeholder'
+                                      )}
+                                    />
+                                  </SelectTrigger>
+                                </FormControl>
+                                <SelectContent>
+                                  {units.map((unit) => (
+                                    <SelectItem value={unit.id}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)